conf/routes

```diff
@@ -75,6 +75,7 @@ GET         /room/:id                                       @controllers.Rooms.s
 POST        /seller/saveroom/:hotelId                       @controllers.Rooms.saveRoom(hotelId: Integer)
 GET         /seller/updateroom/:id                          @controllers.Rooms.editRoom(id: Integer)
 POST        /seller/updateroom/:id                          @controllers.Rooms.updateRoom(id: Integer)
+GET         /seller/updateroom/delete/:id                   @controllers.Prices.delete(id: Integer)
```

Comment:
REST practices dictate that REST Method used should reflect action on the resource. So here is the list:

```
GET  method  - Get resource from WS
POST method - Create resource on WS
PUT method    - Update resource on WS
DELETE method - Delete resource on WS 
```
My recommendation would be that to user

`DELETE /seller/updateroom/:id`